buildscripts/condarecipe.local/meta.yaml

```diff
@@ -34,7 +34,7 @@ requirements:
     - setuptools
     - importlib_metadata       # [py<39]
     # On channel https://anaconda.org/numba/
-    - llvmlite >=0.40.0dev0,<0.40
+    - llvmlite >=0.40.0dev0llvm14,<0.40
```

Comment:
Do we need to split build of llvmlite on aarch64 to use llvm11? Or, can we just say we stay on llvmlite 0.39? This is something we should decide @numba/fpoc

Reply:
xref #8756